### ISSUE TITLE
summary(landsat) error

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#875]
```R
> data(landsat)
> summary(landsat)
Landsat Summary
---------------

* Data file:           /data/archive/landsat/LC80080292014065LGN00
* Time:                2014-03-06 15:02:09
* Spacecraft:          LANDSAT_8
* Header file:         /data/archive/landsat/LC80080292014065LGN00/LC80080292014065LGN00_MTL.txt
* UTM zone:             20 (used for whole image)
* UTM lower left:       287400 easting 4819800 northing (m)
* UTM upper right:      523200 easting 5059500 northing (m)
* Lower left:          -65.629730E 43.500890N
* Lower right:         -62.712880E 43.530780N
* Upper right:         -62.702050E 45.688660N
* Upper left:          -65.728810E 45.656450N
Satellite Summary
-----------------

* Data file:           /data/archive/landsat/LC80080292014065LGN00
* Longitude:           
* Latitude:            
* Time:                2014-03-06 15:02:09 +0000
Error in is.finite(object@data[[i]]) : 
  default method not implemented for type 'list'
```

**Comment by dankelley** [comment]
PS -- check out the weird red backgrounds in the new GH colourizer. I guess it doesn't understand decimal numbers ... ??

**Comment by dankelley** [comment]
The problem is that `landsat` is calling `callNextMethod` and that calls the `satellite` method. So, two thing: (a) I don't like how we get this extra heading name for `Satellite method` and (b) we have the reported error.

**Comment by dankelley** [comment]
Both (a) and (b) fixed in `develop` commit 3f09a5ab04acea42073b462d3fe11da1491a1c5d
